index.html

```diff
@@ -41,12 +41,195 @@ <h1>Hey There. I'm Bright</h1>
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#3" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#4" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#5" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#6" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#7" class="profesional-Art-btm button">See Project</a>
+    </div>
+  </section>
+  <footer>
+    <div class="about-me-footer">
+      <div>
```

Comment:
To make your project semantically accurate and improve your page accessibility, please make sure to use a `section` instead of a `footer` for your About Me section 😉. If you are interested, please check [this website](https://www.orbitmedia.com/blog/website-footer-design-best-practices/) that gives examples of how to create a footer, and [this one](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) to understand more about the `section` element.